### ISSUE TITLE
Rename Member to Membership

### ISSUE THREAD

**Issue by kaichaosun** [basisproject/tracker#108]
When reading the code, AgentID could be `MemberID` and `UserID`, this is confused to me, digging more, Member is more like Membership which maintains the relationship between a user and a company.

I'd like proposing to change from Member to Membership and related concepts.

**Comment by orthecreedence** [comment]
Yes, I agree with you. `Membership` makes a bit more sense than `Member`.
